src/nft/components/profile/view/EmptyWalletContent.tsx

```diff
@@ -45,7 +45,9 @@ export const EmptyWalletContent = () => {
       <EmptyWalletText className={headlineMedium}>
         <Trans>No NFTs in</Trans>&nbsp;{shortenAddress(account ?? '')}
       </EmptyWalletText>
-      <ExploreNFTsButton onClick={() => navigate('/nfts')}>Explore NFTs</ExploreNFTsButton>
+      <ExploreNFTsButton data-testid="nft-explore-nfts" onClick={() => navigate('/nfts')}>
```

Comment:
nit: can data-testid be changed to "nft-explore-nfts-button" for more specificity?